tests/components/totalconnect/common.py

```diff
@@ -1,8 +1,7 @@
 """Common methods used across tests for TotalConnect."""
 from unittest.mock import patch
 
-from total_connect_client.client import TotalConnectClient
-from total_connect_client.const import ArmingState
+from total_connect_client.const import ArmingState, _ResultCode
```

Comment:
It's a bit weird that we import a private name.

Reply:
It was just changed to be private in the total_connect_client package, but there wasn't a way to test without it.  I'll see about making it public again.

Reply:
Are you planning to to resolve this comment in this PR?